arbstate/inbox.go

```diff
@@ -72,12 +73,23 @@ func parseSequencerMessage(ctx context.Context, data []byte, das DataAvailabilit
 			if das == nil {
 				log.Error("No DAS configured, but sequencer message found with DAS header")
 			} else {
-				var err error
-				payload, err = das.Retrieve(ctx, data[40:])
+				cert, err := DeserializeDASCertFrom(bytes.NewReader(data[40:]))
 				if err != nil {
-					log.Error("Reading from DAS failed", "err", err)
+					log.Error("Deserializing data availability cert failed", "err", err)
+					return nil
```

Comment:
We should add an error return to this function, as right now it might crash if it returns nil. Alternatively we could explicitly panic as we do elsewhere.